doc/source/methods/PartialDependenceVariance.ipynb

```diff
@@ -0,0 +1,432 @@
+{
```

Comment:
<p>For future I would suggest putting each paragraph/section in a separate cell for ease of review. Even better if documentation pages are written using <code>.md</code> format and <code>myst-parser</code> . </p><p> </p><ul><li>Missing full stop after the first sentence</li><li>we said that any variable with a flat -&gt; we say that any variable with a flat ?</li><li>continous -&gt; continuous (in latex)</li><li>i(x) should be i(x_1) since the text says "if x_1 is continuous/categorical" ? Actually I'm not sure how to interpret the definition i(x)=F(f_S(x_s)) since the right-hand side has no explicit dependence on x?</li><li>a linear mode -&gt; a linear model</li><li>for the least squares problem formulation I would use the norm markup, i.e. <code>\|</code></li><li>I think the section following needs more references/comments (no need for explicit derivations) so the reader can follow the argument fully if the want, e.g.</li><li class="ql-indent-1">Variance of  beta^ - beta isn't obvious</li><li class="ql-indent-1">why is s^2 an unbiased estimate of sigma^2 isn't obvious</li><li class="ql-indent-1">"once can show that V=..." etc.</li><li class="ql-indent-1">In fact, the whole derivation section of t_i seems like it would be optional. Can you check using e.g. <code>sphinx-design</code> if we could have this as a collapsible toggle section as in Notion? Perhaps it's only possible if re-writing the method in <code>myst</code> markdown.  https://sphinx-design.readthedocs.io/en/furo-theme/dropdowns.html</li><li>be the residuals -&gt; the residuals</li><li>"One can compute the exact f_i(X_i)" - worth reminding the reader that these are the PD functions as the notation here is pretty generic</li><li>each regression coefficients -&gt; each regression coefficient</li><li>the the ones -&gt; the ones</li><li>proposed use the -&gt; propose to use the</li><li>straight forward -&gt; straightforward (all instances in the doc)</li><li>"Consider (X_i, X_j), ...  be any two features -&gt; "Consider any two features (X_i, X_j), ..."</li><li>"a possible feature interactions." -&gt; "a possible feature interaction."</li><li>"we will try to provide" -&gt; "we provide"</li><li>"to say eyeball" -&gt; "to, say, eyeball" or "to eyeball" or even "to decompose"</li><li>&nbsp;"we will first provide" -&gt; "we first provide"</li><li>"is straight forward" -&gt; need grammar improvement, perhaps remove completely the phrase or remove the leading "for"? same for the other two</li><li>"is take" -&gt; "is taken"</li><li>"We will now elaborate" -&gt; "We now elaborate"</li><li>References to Figure 2(a)-(c) - the figure is missing a caption/number</li><li>Would add axis labels to Figures 2 and 3</li><li>"can be attribute to the followings:" -&gt; "can be attributed to the following:"</li><li>"In the following figure displays those the two scenarios" -&gt; rewrite grammar</li><li>For Figure 3 is there a specific value of c in X1=c?</li><li>"Coming back to..." -&gt; better to use active voice like "We now come back to..." since the current sentence finishes abruptly after "Coming back to..." with nothing else said</li><li>For the counter example worth pointing out that "I" is the indicator function (since previously it was used for the identity matrix), or simply use the "1" notation for the indicator function.</li><li>"constants, but" -&gt; "constant, but"</li><li>"can be visualized&nbsp;" -&gt; "are visualized"</li><li><br></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>